app/[locale]/what-is-ether/page.tsx

```diff
@@ -0,0 +1,758 @@
+                </ListItem>
+              </UnorderedList>
+              <LinkWithArrow href="/get-eth/">
+                {t("page-what-is-ether-how-to-buy-eth-description-16")}
+              </LinkWithArrow>
+            </div>
+          </Section>
+
+          <Section
+            id={getId(tocItems[2].url)}
+            className="-scroll-mt-80 space-y-14"
+          >
+            <Image
+              src={ethOrgLogo}
+              alt="Ethereum.org Logo"
+              className="mx-auto max-w-[123px]"
+            />
+            <div className="space-y-6">
+              <h2 id={getId(tocItems[2].url)} className="scroll-mt-28">
+                {tocItems[2].title}
+              </h2>
+              <p>
+                {t.rich(
+                  "page-what-is-ether-how-to-send-and-receive-eth-description-1",
+                  {
+                    strong: Strong,
+                  }
+                )}
+              </p>
+              <p>
+                {t.rich(
+                  "page-what-is-ether-how-to-send-and-receive-eth-description-2",
+                  {
+                    strong: Strong,
+                  }
+                )}
+              </p>
+              <p>
+                {t.rich(
+                  "page-what-is-ether-how-to-send-and-receive-eth-description-3",
+                  {
+                    strong: Strong,
+                    howToUseWallet: (chunks) => (
+                      <Link href="/guides/how-to-use-a-wallet/">{chunks}</Link>
+                    ),
+                  }
+                )}
+              </p>
+              <Alert variant="warning">
+                <AlertEmoji text="💡" />
+                <AlertContent>
+                  <p>
+                    {t.rich("page-what-is-ether-how-to-buy-eth-description-3", {
+                      strong: Strong,
+                    })}
+                  </p>
+                  <p className="mt-2">
+                    {t.rich(
+                      "page-what-is-ether-how-to-send-and-receive-eth-description-4",
+                      {
+                        strong: Strong,
+                      }
+                    )}
+                  </p>
+                  <UnorderedList className="mb-0 mt-2 [&>li]:mb-0.5">
+                    <ListItem>
+                      {t.rich(
+                        "page-what-is-ether-how-to-send-and-receive-eth-description-5",
+                        {
+                          strong: Strong,
+                        }
+                      )}
+                    </ListItem>
+                    <ListItem>
+                      {t.rich(
+                        "page-what-is-ether-how-to-send-and-receive-eth-description-6",
+                        {
+                          strong: Strong,
+                        }
+                      )}
+                    </ListItem>
+                  </UnorderedList>
+                </AlertContent>
+              </Alert>
```

Comment:
(Repeated word-for-word from above... assuming that's not intended cc: @konopkja ?)

Reply:
<img width="993" height="558" alt="Screenshot 2025-10-29 at 10 26 49" src="https://github.com/user-attachments/assets/054f2306-715c-48a8-a268-afd7e6f67bc2" />
 
 
 ohh thanks for flagging this @wackerow ! this is actually my bad and you are right, the sidenotes are supposed to be different:
 
 Only share your Ethereum address with trusted contacts. Since Ethereum is a public ledger, they can view your balance and transactions. They can't access your funds but privacy is recommended.